plugins/action/common/prepare_plugins/prep_108_route_control.py

```diff
@@ -0,0 +1,298 @@
+                                        group_policies.append(group_name)
+
+                                output = template.render(
+                                    MD_Extended=model_data,
+                                    item=model_data["vxlan"]["overlay_extensions"]["route_control"],
+                                    switch=switch['name'],
+                                    group_item=group_policies,
+                                    defaults=default_values)
+
+                                new_policy = {
+                                    "name": unique_name,
+                                    "template_name": "switch_freeform",
+                                    "template_vars": {
+                                        "CONF": output
+                                    }
+                                }
+
+                                model_data["vxlan"]["policy"]["policies"].append(new_policy)
+
+                                if any(sw['name'] == switch['name'] for sw in model_data["vxlan"]["policy"]["switches"]):
+                                    found_switch = next(([idx, i] for idx, i in enumerate(
+                                        model_data["vxlan"]["policy"]["switches"]) if i["name"] == switch['name']))
+                                    if "groups" in found_switch[1].keys():
+                                        model_data["vxlan"]["policy"]["switches"][found_switch[0]]["groups"].append(
+                                            unique_name)
+                                    else:
+                                        model_data["vxlan"]["policy"]["switches"][found_switch[0]]["groups"] = [
+                                            unique_name]
+                                else:
+                                    new_switch = {
+                                        "name": switch["name"],
+                                        "groups": [unique_name]
+                                    }
+                                    model_data["vxlan"]["policy"]["switches"].append(new_switch)
+
+                                if not any(group['name'] == sw_group for group in model_data["vxlan"]["policy"]["groups"]):
+                                    new_group = {
+                                        "name": unique_name,
+                                        "policies": [
+                                            {"name": unique_name},
+                                        ],
+                                        "priority": 500
+                                    }
+                                    model_data["vxlan"]["policy"]["groups"].append(new_group)
+
+            model_data = hostname_to_ip_mapping(model_data)
+        self.kwargs['results']['model_extended'] = model_data
+        return self.kwargs['results']
+
+    def convert_port_number(self, port_number, protocol):
```

Comment:
Is there a reason the model didn't take the well-known cli? Asking because I don't see any of this in the data model. Can you provide a pointer here or via unicast?